clang/test/CIR/CodeGen/globals.c

```diff
@@ -71,6 +71,15 @@ struct { int *x; } p2 = {&p[0]};
 // CHECK: cir.global external @p1 = #cir.global_view<@p> : !cir.ptr<!s32i>
 // CHECK: cir.global external @p2 = #cir.const_struct<{#cir.global_view<@p> : !cir.ptr<!s32i>}> : !ty_22anon2E1122
 
+struct __attribute__((packed)) PackedStruct {
+    char a1;
+    char a2;
+    char a3;
+};
+struct PackedStruct packed[10];
+char *packed_element = &(packed[-2].a3);
+// CHECK: cir.global  external @packed_element = #cir.global_view<@packed, [-2 : i32, 2 : i32]>
```

Comment:
Awesome! Can you please also add a check for the final LLVM output? `clang/test/CIR/CodeGen/abstract-cond.c` has the example on how to add the extra `RUN` line. If for some reason it fails because of other tests in this file, feel free to add the new test to a new different file.